operations/kubernetes/Kyverno.md

```diff
@@ -432,3 +432,14 @@ and [match/exclude](https://release-1-10-0.kyverno.io/docs/writing-policies/matc
 * [False positive audit logs are generated for Validation policy](https://github.com/kyverno/kyverno/issues/3970)
 * [No event is generated in case of mutation policy being applied to a resource](https://github.com/kyverno/kyverno/issues/2160)
 * [Inaccurate annotations are created after applying the policy](https://github.com/kyverno/kyverno/issues/3473)
+* Kyverno failed to verify `tls` certificate for signature verification from CSM 1.6 and above
+
+    ```bash
+    10m         Warning   PolicyViolation   pod/canu-test-6ff597df6d-fbnbd   policy check-image/check-image fail: failed to 
+    verify image registry.local/artifactory.algol60.net/csm-docker/stable/cray-canu/canu-test:1.6.36: 
+    .attestors[0].entries[0].keys: Get "https://registry.local/v2/": tls: failed to verify certificate: x509: certificate 
+    signed by unknown authority
+    ```
+
+    When end user encounters the above issue for signature verification then the solution is to restart the Kyverno pods.
```

Comment:
@pradeepkumargl I do not quite understand this solution ... Was it really tested?

From what I see, Platform TLS certificate is installed by Helm post-install hook. This hook is not executed when pods are simply restarted. It is only executed when Helm chart is installed or upgraded.

Reply:
Hi @mtupitsyn, as discussed, it looks like this was synchronization issue, admission controller started before trust hook injects certificate. After restarting, things were working as expected. This was notices in recent installed systems. I will request Pradeep or Dinesh to attach the test logs here.